src/main/java/org/sonarlint/intellij/analysis/AnalysisCallback.java

```diff
@@ -25,7 +25,7 @@
 import org.sonarlint.intellij.issue.LiveIssue;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonarlint.intellij.issue.LiveIssue'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)